espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/Http/Response/AppiumResponse.java

```diff
@@ -1,27 +1,62 @@
 package io.appium.espressoserver.lib.Http.Response;
 
-import io.appium.espressoserver.lib.Model.Appium;
+import java.util.UUID;
+
+import fi.iki.elonen.NanoHTTPD.Response.Status;
 import io.appium.espressoserver.lib.Model.AppiumStatus;
 
-public class AppiumResponse extends BaseResponse {
-    public AppiumResponse() {
-        setResponse(new Appium());
+@SuppressWarnings("unused")
+public class AppiumResponse {
+    private transient Status httpStatus;
+    private Object value;
+    private AppiumStatus status;
+    private String sessionId;
+    private String id; // Unique Appium transaction ID
```

Comment:
lint does not like such comments style. Usually comments are put right before the line they are related to